components/tls/tls-mpc/src/config.rs

```diff
@@ -5,6 +5,7 @@ static DEFAULT_OPAQUE_RX_TRANSCRIPT_ID: &str = "opaque_rx";
 static DEFAULT_TX_TRANSCRIPT_ID: &str = "tx";
 static DEFAULT_RX_TRANSCRIPT_ID: &str = "rx";
 
+/// Configuration options which are common to both the prover and the notary
```

Comment:
we want to try to use the prover/notary semantics only on the higher level, if possible. Here, it can just be 
"both the leader and the follower"